en/docs/install-and-setup/setup/setting-up-databases/managing-data-growth-and-improving-performance.md

```diff
@@ -11,6 +11,9 @@ WSO2 API Manager provides two methods to do the cleanup.
    
 While the regular cleanup is good for regular housekeeping, **a hybrid approach is recommended for a production environment** that removes all unused token, session, and Registry data. While the regular cleanup will slow down unused token growth, deep cleaning  will take care of the leftover unused data and prevent the tables from continuously growing, impacting performance.   
 
+!!! warn
+    If you have enabled deep cleaning in a server that has low traffic you do not need to enable regular cleaning in these cases. Deep cleaning efficiently removes unused token, session, and Registry data, ensuring optimal performance in production environments. However, during peak windows, regular cleaning may be less effective due to potential query slowdowns. Consider these factors when choosing the appropriate cleanup method.
```

Comment:
Shall we rephrase it as follows?: Deep cleaning is more effective than regular cleaning for servers with low traffic. It removes unused tokens, sessions, and registry data, which can improve performance in the production servers. However, during peak windows, regular cleaning may be less effective due to potential query slowdowns. Choose the appropriate cleanup method based on the server's traffic level.

Reply:
Done!